app/controllers/admin/signature_sheets_controller.rb

```diff
@@ -26,6 +26,12 @@ def show
   private
 
     def signature_sheet_params
-      params.require(:signature_sheet).permit(:signable_type, :signable_id, :required_fields_to_verify)
+      params.require(:signature_sheet).permit(
+        :signable_type,
+        :signable_id,
+        :document_numbers,
```

Comment:
I guess this line was added due fo a confict on this file :thinking:. The field `document_numbers` no longer exists, so we should remove it.